slack_lambda/main.tf

```diff
@@ -19,8 +19,18 @@ locals {
         eventmsg = event['Records'][0]['Sns']['Message']
         try:
           data = json.loads(eventmsg)
-          if data['detailType'] == 'CodePipeline Pipeline Execution State Change':
+          if 'detailType' in data and data['detailType'] == 'CodePipeline Pipeline Execution State Change':
             msgtext = 'auto-terraform:  ' + data['detail']['pipeline'] + ' pipeline ' + data['detail']['state'] + ' with execution ID ' + data['detail']['execution-id']
+          elif 'AlarmName' in data and 'AlarmDescription' in data:
+```suggestion
```

Comment:
oh no whoops i left this line in there 🤦 
```suggestion
```

Reply:
Ah, yep. I was working on a fix and converted to Python f strings for better formatting in 7b7d78a